contracts/concordium-governance-committee-election/src/lib.rs

```diff
@@ -308,7 +333,7 @@ impl From<&State> for ElectionConfig {
             election_manifest: registered_data.election_manifest.clone(),
             election_parameters: registered_data.election_parameters.clone(),
             candidates,
-            guardian_keys,
+            guardian_accounts: guardian_keys,
```

Comment:
Given above suggestion:

```suggestion
            guardian_accounts,
```